src/index.ts

```diff
@@ -38,4 +39,10 @@ program
 	.description('Easy way to add extra things in your sern project')
 	.action(extra);
 
+program
+	.command(publish.name)
+	.description('New way to manage your slash commands')
+	.argument('<name>', 'name of the file without extension')
```

Comment:
we would want to make <name> argument a glob. its a bit repetitive to publish single commands

Reply:
ie: `sern publish src/**/*` works or `sern publish ping` works as well

Reply:
will keep in mind